spec/client_spec.rb

```diff
@@ -92,6 +92,22 @@
         expect(lock_manager).to receive(:sleep).exactly(lock_manager_opts[:retry_count] - 1).times
         lock_manager.lock(resource_key, ttl)
       end
+
+      it 'sleeps at least the specified retry_delay in milliseconds' do
```

Comment:
Hmm, should we maybe stub `rand()` here? Otherwise this test could pass a million times and then fail. Testing randomness is hard...

otoh, the code itself is simple (and "correct by looking at it"), so not sure if the further hassle is worth it :smile: